core/src/avm2/globals/boolean.rs

```diff
@@ -46,10 +46,78 @@ pub fn native_instance_init<'gc>(
 
 /// Implements `Boolean`'s class initializer.
 pub fn class_init<'gc>(
+    activation: &mut Activation<'_, 'gc, '_>,
+    this: Option<Object<'gc>>,
+    _args: &[Value<'gc>],
+) -> Result<Value<'gc>, Error> {
+    if let Some(this) = this {
+        let mut boolean_proto = this
+            .get_property(this, &QName::dynamic_name("prototype").into(), activation)?
+            .coerce_to_object(activation)?;
+        let scope = activation.create_scopechain();
+        let gc_context = activation.context.gc_context;
+        let this_class = this.as_class_object().unwrap();
+
+        boolean_proto.install_dynamic_property(
```

Comment:
(note: I'm getting tempted to attempt making method-on-prototype declarations declarative, like normal class methods)